### ISSUE TITLE
Cannot make owntone running on container in Synology

### ISSUE THREAD

**Issue by SzymonSzumilo** [owntone/owntone-server#1662]
Hallo,
For longer time I am trying to start owntone on Synology (container use). First I cannot get acess to the server on web interface - in I put the adress: port numer but it says only FORBIDDEN. Where shall I set admin password ? In enviromet of the container or in the config file of owntone ? Server Log looks as follows:

httpd: Web interface request to '/' denied: No password set in the config
httpd: Web interface request to '/favicon.ico' denied: No password set in the config

**Comment by ejurgensen** [comment]
I can't help you with Synology containers, but yes, if you want an admin password for OwnTone you can set one in the server config. Otherwise you can use the trusted_networks setting to

**Comment by Coco1780** [comment]
Hello,

I try today to use OwnTone on my synology (Docker).
I got the same error as someone else: "Fatal: getentropy not found"
After some research the problem is in the "libgcrypt" library which is compiled with "--enable-random=getentropy", I compiled my own version with "--enable-random=linux".
I replaced the lib in the container and then it works.

I'm posting this information here for those looking for an answer like me.
